templates/prometheus.yaml

```diff
@@ -80,6 +80,38 @@ spec:
                   regex: '(.*):10250'
                   replacement: '${1}:1338'
                   target_label: __address__
+            - job_name: 'kubernetes-calico-node'
```

Comment:
can you share an example of the prometheus targets to validate that this job is currently working as expected ?

Reply:
```
up{job="kubernetes-calico-node"}


up{endpoint="bird-metrics",instance="10.0.128.213:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-128-213.us-west-2.compute.internal",pod="calico-node-rxq45"} | 1
up{endpoint="bird-metrics",instance="10.0.128.4:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-128-4.us-west-2.compute.internal",pod="calico-node-72s98"} | 1
up{endpoint="bird-metrics",instance="10.0.130.134:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-130-134.us-west-2.compute.internal",pod="calico-node-hhfvb"} | 1
up{endpoint="bird-metrics",instance="10.0.130.183:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-130-183.us-west-2.compute.internal",pod="calico-node-q9l2p"} | 1
up{endpoint="bird-metrics",instance="10.0.192.82:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-192-82.us-west-2.compute.internal",pod="calico-node-gbs7q"} | 1
up{endpoint="bird-metrics",instance="10.0.192.92:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-192-92.us-west-2.compute.internal",pod="calico-node-766wb"} | 1
up{endpoint="bird-metrics",instance="10.0.194.217:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-194-217.us-west-2.compute.internal",pod="calico-node-xg47c"} | 1
up{endpoint="felix-metrics",instance="10.0.128.213:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-128-213.us-west-2.compute.internal",pod="calico-node-rxq45"} | 1
up{endpoint="felix-metrics",instance="10.0.128.4:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-128-4.us-west-2.compute.internal",pod="calico-node-72s98"} | 1
up{endpoint="felix-metrics",instance="10.0.130.134:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-130-134.us-west-2.compute.internal",pod="calico-node-hhfvb"} | 1
up{endpoint="felix-metrics",instance="10.0.130.183:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-130-183.us-west-2.compute.internal",pod="calico-node-q9l2p"} | 1
up{endpoint="felix-metrics",instance="10.0.192.82:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-192-82.us-west-2.compute.internal",pod="calico-node-gbs7q"} | 1
up{endpoint="felix-metrics",instance="10.0.192.92:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-192-92.us-west-2.compute.internal",pod="calico-node-766wb"} | 1
up{endpoint="felix-metrics",instance="10.0.194.217:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-194-217.us-west-2.compute.internal",pod="calico-node-xg47c"} | 1
```